tests/preprocessing/image/test_image_resize.py

```diff
@@ -0,0 +1,251 @@
+# MIT License
```

Comment:
I think a new call to pytest for tests/preprocessing/image is needed in run_tests.sh after line 39.